jobs/README.md

```diff
@@ -4,6 +4,19 @@ In this section you can find some job openings listed by our community members.
 
 __Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
 
+## Βlockchain / Smart contract Engineer (Remote)
+
+Good evening!
+
+I ([maninak](https://app.slack.com/client/T06RGB4BU/C06RGB4DQ/user_profile/U2F3S7MMW)) am forwarding a job listing for a friend and colleague based in London. From what I understand it's about a promising new project in the crypto space, for which an international team is being set up and with some big names in the industry behind it, therefore it's a bit hush-hush in the beginning, hence the brief listing.
+
+> We are looking for developers with experience in web3 projects and knowledge of ETH/Solidity, smart contracts and Solana/Rust (optional).
```

Comment:
@maninak I would recommend adding some more formal details about this job opening. Maybe a link to the job ad, the company itself or anything else that someone could consider about this ad.